examples/run_examples.jl

```diff
@@ -0,0 +1,40 @@
+#  Copyright 2018, Iain Dunning, Joey Huchette, Miles Lubin, and contributors
+#  This Source Code Form is subject to the terms of the Mozilla Public License,
+#  v. 2.0. If a copy of the MPL was not distributed with this file, You can
+#  obtain one at http://mozilla.org/MPL/2.0/.
+
+#=
+    This script runs all the JuMP examples.
+    julia --project=examples run_examples.jl
+=#
+
+using Test
+
+const EXAMPLES = [
```

Comment:
To get rid of an extra list that needs to be maintained, we could also run all `.jl` files in the examples directory.